src/um/d3d11tokenizedprogramformat.rs

```diff
@@ -0,0 +1,1191 @@
+// Copyright © 2015-2017 winapi-rs developers
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+use shared::minwindef::{DWORD, UINT};
+ENUM!{enum D3D10_SB_TOKENIZED_PROGRAM_TYPE {
+    D3D10_SB_PIXEL_SHADER = 0,
+    D3D10_SB_VERTEX_SHADER = 1,
+    D3D10_SB_GEOMETRY_SHADER = 2,
+    D3D11_SB_HULL_SHADER = 3,
+    D3D11_SB_DOMAIN_SHADER = 4,
+    D3D11_SB_COMPUTE_SHADER = 5,
+    D3D11_SB_RESERVED0 = 0xFFF0,
+}}
+pub const D3D10_SB_TOKENIZED_PROGRAM_TYPE_MASK: DWORD = 0xffff0000;
+pub const D3D10_SB_TOKENIZED_PROGRAM_TYPE_SHIFT: DWORD = 16;
+#[inline]
+pub fn DECODE_D3D10_SB_TOKENIZED_PROGRAM_TYPE(VerTok: DWORD) -> DWORD {
+    (((VerTok & D3D10_SB_TOKENIZED_PROGRAM_TYPE_MASK) >> D3D10_SB_TOKENIZED_PROGRAM_TYPE_SHIFT) as D3D10_SB_TOKENIZED_PROGRAM_TYPE)
```

Comment:
It looks like quite a few lines here exceed the 99 chars limit.
> ## [Newlines and indentation](https://github.com/retep998/winapi-rs/blob/0.3/CONTRIBUTING.md#newlines-and-indentation)
> - The maximum line length for `winapi-rs` is 99, and is strictly enforced.

Reply:
What should be the formatting style for lines that have been broken? default rustfmt?